orchagent/routeorch.cpp

```diff
@@ -688,6 +704,29 @@ void RouteOrch::doTask(Consumer& consumer)
                     {
                         nhg = NextHopGroupKey();
                     }
+                    else if (srv6_nh == true)
+                    {
+                        string ip;
+                        if (ipv.empty())
+                        {
+                            ip = "0.0.0.0";
+                        }
+                        else
+                        {
+                            SWSS_LOG_ERROR("For SRV6 nexthop ipv should be empty");
+                            continue;
```

Comment:
missing `it = consumer.m_toSync.erase(it);` ?

Reply:
Done, for the invalid case.